### PR TITLE
Add reactbase aa1 to master

### LINKED ISSUE #1: There should be standard files so that the project can be initiated
**Is your feature request related to a problem? Please describe.**
Working with a blank page without reference points is a pain.

**Describe the solution you'd like**
Add a package.json, navbar and footer so that the basics are established.

### PR DESCRIPTION
**Pull request by petschna**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently there are no standard files to start the project. #1

**What is the new behavior (if this is a feature change)?**
- Now there are standard files to kick off the start of the project. Closes #1

### CODE REVIEW

**Review by Aichnerc — APPROVED** [review]
LGTM, so what am I getting paid?

**Review by petute — APPROVED** [review]
Should be ok to merge.

File: src/components/pages/PageRoot.js [hunk 1]
```diff
@@ -7,17 +7,17 @@ import React from "react";
 import { MDBFreeBird, MDBCol, MDBRow, MDBCardBody, MDBIcon } from "mdbreact";
 
 //> Components
+
```
Comment (kleberbaum) [inline src/components/pages/PageRoot.js]:
```suggestion
```
Would be better.

**Review by kleberbaum — CHANGES_REQUESTED** [review]
Should be ok to merge after a few changes.

File: src/Routes.js [hunk 2]
```diff
@@ -8,13 +8,13 @@ import { Route, Switch } from "react-router-dom";
 /**
  * HomePage: A basic template page
```
Comment (kleberbaum) [inline src/Routes.js]:
```suggestion
 * PageRoot: A basic template page
```

**Review by kleberbaum — CHANGES_REQUESTED** [review]
Should be ok to merge after a few changes.

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Pull request merged**